manager/assets/modext/sections/security/profile/update.js

```diff
@@ -208,38 +252,60 @@ Ext.extend(MODx.panel.UpdateProfile,MODx.FormPanel,{
     },
     getItemsRight: function (config) {
         var items = [{
+            id: 'modx-user-country'
+            ,name: 'country'
+            ,fieldLabel: _('user_country')
+            ,xtype: 'modx-combo-country'
+            ,anchor: '100%'
+            ,value: ''
```

Comment:
Please remove the value. Maybe then #14732 will be solved with that.

Reply:
This does not help, although the bug now works differently :)

![country_2](https://user-images.githubusercontent.com/12523676/64629815-f496ec00-d404-11e9-9410-a993c6227ed9.gif)

Reply:
Without `value: ''` the default value is '0' there. Where is that panel filled?

Reply:
I do not quite understand your question. If you delete `value: ''`, then when updating (sometimes), instead of an empty string, 0 appears.
This is relevant for this files (with or without this PR): https://github.com/modxcms/revolution/blob/3.x/manager/assets/modext/widgets/security/modx.panel.user.js#L600 and https://github.com/modxcms/revolution/blob/3.x/manager/assets/modext/sections/security/profile/update.js#L237.
This is relevant for this sections of the manager panel: `/manager/?a=Security/User/Update&id=1` and `/manager/?a=security/profile` .

Reply:
Do you see the flickering value just before the blank value occurs? It shows `DZ`, but the keys i.e. in core/lexicon/country/en.inc.php are lowercase. Is that user created in MODX? How did you get the `DZ` value in that field?